tests/sanity/ignore-2.11.txt

```diff
@@ -0,0 +1,4 @@
+plugins/modules/redhat_csp_download.py validate-modules:undocumented-parameter
```

Comment:
Can we update the module to avoid as many ignored tests as possible?

Reply:
I was ignoring some issues with Python 3.9 https://pastebin.com/7YBhVQ9v